docs/policy.md

```diff
@@ -263,3 +263,58 @@ In an example below admin has access to all methods in all paths except `delete`
   resource:
     path: /v2.0/restricted_method.*
 ```
+
+## Attach policy
+
+Attach policy defines allowed values for a relation property
+(a resource "attaches" to another resource by having a relation to it).
+Such policy defines restrictions for a property between a _source_ and a _target_ resource.
+The syntax is slightly different than in the case of a standard policy:
+
+- id, principal, resource, tenant_id: as in the case of a standard policy
+- action: Must be "allow"
+- effect: If the condition in the policy allows for the access, or denies it.
+  Possible values (case insensitive, default is "allow"):
+    - "allow": policy denies if source or target conditions are false
+    - "deny": policy denies if source and target conditions are true
+- resource : Like in the case of a standard policy, but considers the source resource
+- relation_property: Name of the relation property the policy is applied to
```

Comment:
can we wildcard this value? Would be nice to have some defaults.

Reply:
Added possibility to specify a wildcard `relation_property`.